opensrp-app/src/main/java/org/smartregister/util/JsonFormUtils.java

```diff
@@ -407,23 +407,23 @@ public static void addObservation(Event e, JSONObject jsonObject) {
         if (StringUtils.isBlank(value)) { return; }
 
         String type = getString(jsonObject, AllConstants.TYPE);
-        String entity =  getString(jsonObject, OPENMRS_ENTITY);
+        String entity =  CONCEPT;
```

Comment:
Doesn't this assume that the `openmrs_entity` has to be type concept? It could be empty, right?